sdk/communication/azure-communication-common/src/main/java/module-info.java

```diff
@@ -4,13 +4,11 @@
 module com.azure.communication.common {
 
     requires transitive com.azure.core;
+    requires com.azure.json;
 
     // public API surface area
     exports com.azure.communication.common;
 
-    opens com.azure.communication.common
```

Comment:
The migration guide said to replace `opens` statements for Jackson with a similar one for azure-json. I think we should add that or other pipelines for libraries that use this package might break.